### PR TITLE
Ncgames 8

### PR DESCRIPTION
**Pull request by gracefaz**
Complete #8: /api/reviews, passing 200 and 404 tests.

### CODE REVIEW

File: models/reviews.models.js [hunk 1]
```diff
@@ -35,3 +35,21 @@ exports.updateVotes = (reviewId, incVotes) => {
       }
     });
 };
+
+// ALMOST WORKS BUT JUST MISSING THE comment_count KEY.
```
Comment (Rosem10) [inline models/reviews.models.js]: Remove please!

File: models/reviews.models.js [hunk 2]
```diff
@@ -35,3 +35,21 @@ exports.updateVotes = (reviewId, incVotes) => {
...
+// exports.fetchReviews = () => {
+//   return db.query(`SELECT * FROM reviews`).then((reviews) => {
+//     console.log(reviews.rows, "<--- reviews.rows");
+//     return reviews.rows;
+//   });
+// };
+
+exports.fetchReviews = (sortBy = "date", order = "DESC") => {
+  let queryStr = `SELECT reviews.owner, reviews.title, reviews.review_id, reviews.category, reviews.review_img_url, reviews.created_at, reviews.votes,
```
Comment (Rosem10) [inline models/reviews.models.js]: You could make this dry-er by using reviews.* instead of writing each column name

**Review by Rosem10 — APPROVED** [review]

File: models/reviews.models.js (hunk 1 above)
Comment (gracefaz) [reply models/reviews.models.js]: Already removed before pushing so not sure why its here!

File: models/reviews.models.js (hunk 2 above)
Comment (gracefaz) [reply models/reviews.models.js]: Can I merge after fixing this?

**Pull request merged**